src/lib/Dashboard/SectionInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace EzSystems\HybridPlatformUi\Dashboard;
+
+interface SectionInterface
```

Comment:
This is the same as `TabInterface`. They are small, but could we reduce duplication by having a common top level interface, or a `Renderable` trait, maybe?